features/upsert.feature

```diff
@@ -31,7 +31,7 @@ Feature: Upsert Cache Time
       {
         "path": "/some/other/path",
         "etag": "a-different-etag",
-        "collection_id": 999,
+        "collection_id": "test-1a19e3462937d85804752375daa00ba41d1b6625d396f21000e3c4571ebf2606",
```

Comment:
In this test, we want to make sure that the whole resource is updated, so this is a bad idea. This should be completely different to the original one.

Reply:
addressed

Reply:
This has been done the other way around: this should be the `updatedcollectionid-` collection, the original should be the `test-`. Otherwise, it looks confusing.